vmware/ngc/NGC-AdapterManager/src/test/java/org/opensds/vmware/ngc/adapters/opensds/OpenSDSTest.java

```diff
@@ -0,0 +1,108 @@
+package org.opensds.vmware.ngc.adapters.opensds;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.Properties;
+
+import org.junit.jupiter.api.Test;
+import org.opensds.vmware.ngc.models.ALLOC_TYPE;
+import org.opensds.vmware.ngc.models.ATTACH_MODE;
+import org.opensds.vmware.ngc.models.ATTACH_PROTOCOL;
+import org.opensds.vmware.ngc.models.ConnectMO;
+import org.opensds.vmware.ngc.models.HOST_OS_TYPE;
+import org.opensds.vmware.ngc.models.POOL_TYPE;
+import org.opensds.vmware.ngc.models.SnapshotMO;
+import org.opensds.vmware.ngc.models.StoragePoolMO;
+import org.opensds.vmware.ngc.models.VolumeMO;
+
+class OpenSDSTest {
+	private String openSdsIP;
+	private String userName;
+	private String password;
+	private int port;
+	private String esxIP;
+	private String esxIQN;
+
+	@Test
+	void ITtestOpenSDS() throws Exception {
+		OpenSDS osds = new OpenSDS("OpenSDS");
+		readDefaultConfig();
+		osds.login(openSdsIP, port, userName, password);
+		osds.getDeviceInfo();
+		List<StoragePoolMO> pools = osds.listStoragePools();
+		StoragePoolMO pool = SelectBlockPool(pools);
+		if (pool == null) {
+			fail("List Pool  Failure");
+		}
+		VolumeMO volume = osds.createVolume("test_volume", "test volume creation", ALLOC_TYPE.THIN, 1024 * 1024 * 1024,
```

Comment:
Can we create Constants for gb conversion and also for sleep time? though it is unit test

Reply:
Good suggestion, Thanks. Modified